flask/todo_app/services/email_service.py

```diff
@@ -1,16 +1,12 @@
 from flask import render_template
 from flask_mail import Message
 
-from todo_app.extensions.mail import mail
+from todo_app.extensions.mail_ext import mail
 
-DEFAULT_SENDER = 'Team Todo <help@todo.com>'
+DEFAULT_SENDER = 'Team Todo <help@example.com>'
 
 
 def send_reset_password(user, password):
-    msg = Message("Reset Your Password",
-                  sender=DEFAULT_SENDER,
-                  recipients=[user.email])
-
+    msg = Message("Reset Your Password", sender=DEFAULT_SENDER, recipients=[user.email])
```

Comment:
what is your rule for single line vs multiple line?  sometimes I agree w/ you and others I am not sure.